<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartClass Cognitive Services</string>
  </property>
  <widget class="QGraphicsView" name="linerchart">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>700</width>
     <height>300</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="pic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="piechart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>450</width>
     <height>350</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>320</y>
     <width>550</width>
     <height>350</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="getEmotionButton">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>685</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>get emotion</string>
   </property>
  </widget>
  <widget class="QPushButton" name="speechButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>685</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>input voice</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>685</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
